### PR TITLE
updatedReadme

### PR DESCRIPTION
**Pull request by Amrutha1101**
## What is the change?
I've updated readme file -> added PSoC to open-source event list 

## Related issue?
close: #1054

## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

- [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
- [x] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/BlogSite/pulls) for the same update/change?
- [x] Did you lint your code before making the Pull Request?
- [x] Have you made corresponding changes to the documentation?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -190,7 +190,11 @@ MAILGUN_API_KEY = <your-api-key>
 
 - GirlScript Summer of Code is the 3 month long Open Source program  conducted by GirlScript Foundation, started in 2018, with an aim to help beginners get started with Open Source Development while encouraging diversity.
 
-<img align="right" src="assets/event4.png" width = "120" height = "100"><b>4. Let's Grow More Summer of Code</b>
```
Comment (Pranav016) [inline README.md:193]: Why did you remove LGM-SOC from the list ?

File: README.md [hunk 2]
```diff
@@ -190,7 +190,11 @@ MAILGUN_API_KEY = <your-api-key>
...
+<img align="right" src="assets\event4.png" width = "120" height = "100"><b>4. PClub Summer of Code</b>
+
+- PSOC( PClub Summer of Code ) is the 2 month long Open Source program during summers conducted by Programming Club, UIET, with an aim to help beginners get started with Open Source Development while encouraging Diversity .
+
+<img align="right" src="assets\event5.png" width = "120" height = "100"><b>5. Let's Grow More Summer of Code</b>
 
 - Let's Grow More Summer of Code is a one month long Open Source program organized by LetsGrowMore Community to encourage student participation in open source software development under the guidance of mentors from the open source community.
```
Comment (Pranav016) [inline README.md:199]: You put PSoC between LGM-SOC heading and info ?

**Review by Pranav016 — CHANGES_REQUESTED** [review]
revert changes

File: README.md (hunk 2 above)
Comment (Amrutha1101) [reply README.md:199]: Yes PSoc as the 4th event and LGMSoc as the th event

Comment (Amrutha1101) [reply README.md:199]: Lgmsoc as the 5th event**

Comment (Pranav016) [reply README.md:199]: But you removed lgm-soc heading and image

**Review by Pranav016 — APPROVED** [review]

**Comment by Pranav016** [comment]
Are you working under LGM-SOC ?

**Comment by Pranav016** [comment]
Also images got messed up, check readme once
image

**Comment by Amrutha1101** [comment]
> Are you working under LGM-SOC ?

yes

**Comment by Amrutha1101** [comment]
> Also images got messed up, check readme once
> image

Is that so?? But it's merged??

**Comment by Pranav016** [comment]
> > Also images got messed up, check readme once
> > image
> 
> Is that so?? But it's merged??

You can make a new PR
